tests/envs/test_sawyer_envs.py

```diff
@@ -75,29 +76,29 @@ def run_pick_and_place(*_):
     algo.train()
 
 
-def test_reacher():
-    """Testing for reacher."""
+class TestSawyerEnvs(unittest.TestCase):
+    def test_reacher(self):
+        """Testing for reacher."""
 
-    env = ReacherEnv()
-    for i in range(9999):
-        env.render()
-        action = env.action_space.sample()
-        next_obs, reward, done, _ = env.step(action)
-    env.reset()
-    env.close()
+        env = ReacherEnv()
+        for i in range(9999):
+            env.render()
+            action = env.action_space.sample()
+            next_obs, reward, done, _ = env.step(action)
+        env.reset()
+        env.close()
 
+    test_reacher.render = True
 
-def test_pnp():
-    """Testing for pick and place."""
+    def test_pnp(self):
+        """Testing for pick and place."""
 
-    env = PickAndPlaceEnv()
-    for i in range(9999):
-        env.render()
-        action = env.action_space.sample()
-        env.step(action)
-    env.reset()
-    env.close()
+        env = PickAndPlaceEnv()
+        for i in range(9999):
+            env.render()
```

Comment:
You can test the environment variable `CI` to avoid calling render but still run the test